lib/assets/javascripts/twine.js.coffee

```diff
@@ -283,6 +283,15 @@ Twine.bindingTypes =
         $(node).attr(key, value || null)
       lastValue = newValue
 
+  'bind-prop': (node, context, definition) ->
+    fn = wrapFunctionString(definition, '$context,$root', node)
+    lastValue = {}
+    return refresh: ->
+      newValue = fn.call(node, context, rootContext)
+      for key, value of newValue when lastValue[key] != value
+        $(node).prop(key, value || null)
```

Comment:
Regarding `value || null`: not sure if this if this is what the behaviour should be. For `bind-attribute` it makes sense because setting, say, the `disabled` attribute to `false` will make it look like `disabled="false"` and still be disabled, when you'd want it to enabled. For checked, at least, false will uncheck it the same as null would.